### DISCUSSION TITLE
Question on how to integrate the Alfen Eve Single Pro-line with NHC

### DISCUSSION THREAD

**Discussion by raftheunis87** [joleys/niko-home-control-II#182] [Q&A] [answered]
Hey,

I noticed in the changelog for version 4.5.0 that there is support for Alfen Eve. 

I have such a charger for my car, but it's not hooked up to NHC. Do I need a special module for this? Cause this is new for me.
And what are the possibilities then?

**Comment by tijsverkoyen** [comment, marked as answer]
I don't own a charger so I don't know for sure. But I think you can hook it up through the Niko Home Control programming software: https://guide.niko.eu/en/umnhc2/lv/alfen-and-niko-home-control?l=en

**Comment by raftheunis87** [comment]
Thanks a lot! Looks like you also need the wireless smart hub to connect your P1 meter. Better to connect my Alfen to HA directly then to have more control I guess...
